olm-catalog/serverless-operator/hack/002-serving-drop-unsupported-fields.patch

```diff
@@ -31,8 +31,8 @@ index aa8f3e459..4b4f9b580 100644
                config:
                  additionalProperties:
                    additionalProperties:
-@@ -127,13 +108,8 @@ spec:
-                       type: object
+@@ -770,13 +751,8 @@ spec:
+                       type: array
                ingress:
                  description: The ingress configuration for Knative Serving
 +                x-kubernetes-preserve-unknown-fields: true # To allow for some fields we've deleted.
```

Comment:
perhaps adding which fields might help the reader?

Reply:
or a link for additional context.

Reply:
This pre-existed, not part of this PR, let's open another issue for this.

Reply:
Ok, reviewing a patch of patches isn't easy and I thought it was introduced here.